doc/source/operations/upgrading-openstack.rst

```diff
@@ -35,6 +35,64 @@ Notable changes in the |current_release| Release
 There are many changes in the OpenStack |current_release| release described in
 the release notes for each project. Here are some notable ones.
 
+RabbitMQ SLURP upgrade
+----------------------
+
+Because this is a SLURP upgrade, RabbitMQ must be upgraded manually from 3.11,
+to 3.12, then to 3.13 on Antelope before the Caracal upgrade. This upgrade
+should not cause an API outage (though it should still be considered "at
+risk").
+
+There are two prerequisites:
+
+1. Kolla-Ansible should be upgraded to the latest version. Assuming a typical
+   production layout, execute the following from the kolla-ansible source
+   directory:
+
+   .. code-block:: bash
+
+      git fetch && git pull && ../../venvs/kolla-ansible/bin/pip install .
+
+2. The RabbitMQ container image tag must be equal to or newer than
+   ``20240823T101942``. Check the timestamps in
+   ``etc/kayobe/kolla-image-tags.yml``.
+
+Once complete, upgrade RabbitMQ:
+
+.. code-block:: bash
+
+   kayobe overcloud container image pull -kt rabbitmq
```

Comment:
```suggestion
   kayobe overcloud service configuration generate --node-config-dir /tmp/ignore -kt none
```
This container image pull will only bring in the 3.11 image that we won't even use. Speaking to @Alex-Welsh in the office, he put this here to make sure that the new local kolla config is generated.
I suggest we switch to a more explicit command for this, it's the same workaround we've used elsewhere when needing to generate kolla config while using ``kayobe kolla ansible run``.

Can't wait for Caracal, where we get this patched: https://review.opendev.org/c/openstack/kayobe/+/896394

Reply:
Done